packages/orbit-components/src/OrbitProvider/index.tsx

```diff
@@ -22,6 +23,17 @@ import type { Props } from "./types";
  *
  */
 const OrbitProvider = ({ theme, children, useId }: Props) => {
+  React.useEffect(() => {
+    const ID = "orbit-theme-css-vars";
+    const style = document.createElement("style");
+    style.setAttribute("id", ID);
+    style.innerText = tokensToCssVars({ tokens: theme.orbit }).replace(/\s/g, "");
```

Comment:
This doesn't work. It needs a selector.

```suggestion
    style.innerText = `
    :root {
      ${tokensToCssVars({ tokens: theme.orbit }).replace(/\s/g, "")}
    }
    `;
```